app/models/spree/credit_card_decorator.rb

```diff
@@ -25,4 +28,16 @@ def has_payment_profile? # rubocop:disable Style/PredicateName
   def save_requested_by_customer?
     !!@save_requested_by_customer
   end
+
+  private
+
+  def default_missing?
+    user.credit_cards.where(is_default: true).none?
+  end
+
+  def ensure_default
```

Comment:
I had to read the method's implementation to actually understand what it is about. Is there a better name? What about `ensure_single_default_card`?